apollo/GameOver.tsx

```diff
@@ -90,9 +98,12 @@ const pickWinningPlayer = (
   if (condition.type === WinCriteria.DefeatAmount) {
     return (
       condition.players?.length ? condition.players : activeMap.active
-    ).find(
-      (playerID) =>
-        activeMap.getPlayer(playerID).stats.destroyedUnits >= condition.amount,
+    ).find((playerID) =>
+      !condition.optional
+        ? activeMap.getPlayer(playerID).stats.destroyedUnits >= condition.amount
+        : !condition.completed?.has(playerID) &&
+          activeMap.getPlayer(playerID).stats.destroyedUnits >=
+            condition.amount,
     );
```

Comment:
I think this could be a bit more readable, like this:

```tsx
.find((playerID) =>
  (!condition.optional || !condition.completed?.has(playerID)) && activeMap.getPlayer(playerID).stats.destroyedUnits >= condition.amount,
);
```

Reply:
This is way better 👍. Thanks for the suggestion!

Reply:
This is done in d2a5c9b7729fe01c0909fa1f2c4940e58de270d6